### ISSUE TITLE
1.16.5 crash on world enter

### ISSUE THREAD

**Issue by ghost** [Asek3/Rubidium#147]
Playing via a remote ShadowPC (Windows 10) on a Macbook Air M1 with no other mods installed.

The game hard crashes as soon as I enter a new world with the exit code -1073741819

Turning off all Rubidium in-game settings changes nothing.
Starting Minecraft with no JVM arguments changes nothing.

[debug.log](https://github.com/Asek3/Rubidium/files/8865741/debug.log)
[latest.log](https://github.com/Asek3/Rubidium/files/8865742/latest.log)

Every single Forge port of Sodium I have tried crashes this way on my ShadowPC except for Chlorine (which is abandoned). But, surprisingly, Sodium on Fabric works perfectly lol

**Comment by Asek3** [comment]
oh shit supporting M1 is so annoying

**Comment by ghost** [comment]
Just want to point out, this is an issue with the actual VM ShadowPC running Windows 10. My M1 mac is just streaming the ShadowPC so it shouldn't matter

**Comment by ghost** [comment]
Upon seeing yesterday's update, I installed it in hope that this issue was resolved and upon doing so I noticed the following:

Whenever I spawn on land, the game immediately hard crashes.
Whenever I spawn in an ocean biome however, I can walk around for 2-6 seconds before it hard crashes.

Also, enabling/disabling any combination of settings changes nothing. Installing Oculus also changes nothing.

I hope this helps!

**Comment by ghost** [comment]
I feel like somehow the port from Fabric to Forge has made this mod incompatible with VMs. Because Sodium on Fabric runs great on my ShadowPC

**Comment by Asek3** [comment]
Is there crash-report?

**Comment by ghost** [comment]
The game crashes so hard there's no crash log, so all I could get are these two logs:

[debug.log](https://github.com/Asek3/Rubidium/files/9167203/debug.log)
[latest.log](https://github.com/Asek3/Rubidium/files/9167204/latest.log)

Also recorded a video for you:

https://user-images.githubusercontent.com/17407989/180428091-ada0d648-1c3e-4ff4-9423-7fe9e053ab93.mp4

**Comment by papillo12** [comment]
Did you install the mods of Magnesium/Rubidium Extras and Magnesium/Rubidium dynamic lights using curseforge or gdlauncher or something similar?

**Comment by ghost** [comment]
No, what you are seeing is a fresh profile of 1.16.5 forge v36.2.35 with only Rubidium installed and nothing else.

**Comment by ghost** [comment]
Also, I am not using gdlauncher or curseforge or any other launcher. Just the vanilla launcher and I drag/drop the .jar file into the "mods" folder

**Comment by ghost** [comment]
So, the newest update hasn't fixed the issue, but I have done some research and testing on my own and get this:

Rubidium is a fork of Magnesium, right? So I tested all of the old Magnesium versions and discovered that only "sodium reforged 1.3.2" runs normally. All other versions seem to be causing this crash. Very weird

**Comment by iMonZ** [comment]
> oh shit supporting M1 is so annoying

Actually not. Apple is just dropping Support for Legacy Technology.
Using Zink to Support Vulkan would increase the Framerate drastically, since we could use modern technology here without rewriting everything.
Zink+MoltenVK on macOS and only Zink on every other Plattform.
